packages/experimental/NativeFontMetrics/src/NativeFontMetrics.ios.ts

```diff
@@ -0,0 +1,10 @@
+import { NativeModules } from 'react-native';
+import { ScaleFactors, TextStyle } from './NativeFontMetrics.types';
+
+export const NativeFontMetrics = NativeModules.FRNFontMetrics;
+
+interface NativeFontMetricsInterface {
```

Comment:
I like this is an interface. Should make it easier to convert this to a Turbo Module in the future (if we need to)